### ISSUE TITLE
Comparison diagram isn't very informative and makes pest look bad.

### ISSUE THREAD

**Issue by AGausmann** [pest-parser/site#5]
- It only lists two competitors, and only one of those is in the parser family. Benchmarks against more of the common parsers like combine and lalrpop would also be useful.

- Pest is the listed as the slowest solution, with barely any explanation as to how it could be better. Making excuses for why they run faster is a weak argument; highlighting Pest's strengths is a lot better.

**Comment by dragostis** [comment]
Thanks for the raising this issue. Here's my stance on the matter: both parsers listed in the comparison offer great performance and pest's goal is not to compete with them, but merely to show how pest can offer performance in the same order of magnitude.

Now, things will be different in 3.0. I have rewritten the generator and I'm working on the optimizer with great results.

**Comment by LeoDog896** [comment]
I can't find any new benchmarks on pest - is there any updated information from the performance benchmarks mentioned above?

**Comment by lwandrebeck** [comment]
There's a nom_benchmark repository in geal/gcouprie (nom's author) that could be used (it runs nom, pest and one or two others as far as i can remember), but it's quite outdated, and needs some work.
Otherwise, maybe we should have our own benchmark suite to keep a good tracking of pest perfs throughout time ? I'll be glad to give a hand on that front.

**Comment by LeoDog896** [comment]
Good idea! It'll be nice if we can get an official repo created for that.

**Comment by lwandrebeck** [comment]
There's a nom_benchmark repository in geal/gcouprie (nom's author) that could be used (it runs nom, pest and one or two others as far as i can remember), but it's quite outdated, and needs some work.
Otherwise, maybe we should have our own benchmark suite to keep a good tracking of pest perfs throughout time ? I'll be glad to give a hand on that front.
Edit : https://github.com/rust-bakery/parser_benchmarks for reference
